_layouts/project_extended.html

```diff
@@ -0,0 +1,92 @@
+---
+layout: page
+block: project
+---
+
+<section class="project__image">
+    <img src="{{ page.image }}">
+</section>
+
+<section>
+    {% include centerheading.html title="Project Overview: " %}
+    
+    <div class="row">
+        <div class="col-xs-12 col-md-6">
+	<h5><b>Goal:</b></h5>
+        </div>
+        <div class="col-xs-12 col-md-6">
+	
+        </div>
+    </div>
+    
+    <div class="row2">
+        <p class="about__description col-xs-12 col-md-6">
+ The goal of <a href="https://www.indyautonomouschallenge.com">Indy Autonomus Challenge</a> is to implement a software stack capable of operating a Formula One style car in a head to head competition against teams all over the world in a setting similar to the Indy 500. The final race is scheduled for October 23, 2021 at the Indiapolis Motor Speedway, however preceding that there are multiple qualifing rounds including the writing of a white paper, a demonstartion of basic operation, and competing in a simulated race.
+        <div class="col-xs-12 col-md-6">
+           <a href="https://indyautonomouschallenge.com/timeline"> <img class="about__group-image" src="/assets/images/projects/2020/indy-timeline.jpg"></a>
```

Comment:
We need to remove the header (goal) and resize the image to make it larger

Reply:
Please note that the html files in layouts directory are previously just that, layouts. These are meant to be used as bases for actual dynamic content to be loaded, in this case, in the markdown files. please refer to the _pages/projects as an example of how this is done.

Fiddling with layouts like this may result in unintended consequences